app/models/manager_refresh/inventory/middleware_manager.rb

```diff
@@ -0,0 +1,64 @@
+module ManagerRefresh::Inventory::MiddlewareManager
+  extend ActiveSupport::Concern
+  include ::ManagerRefresh::Inventory::Core # if full name is not specified, Travis raises errors.
```

Comment:
I tried `include Core` locally and it does not throw errors... but a FQNN does not hurt.. so, ok